release/4.6.yaml

```diff
@@ -13,6 +13,8 @@ rules:
     refuse:
       - do-not-merge/hold
       - needs-rebase
+    require:
+      - backport-risk-assessed
```

Comment:
how is the code handling the other required labels today? (approved, lgtm'd)

Assuming they are currently hardcoded, can we instead get them all configured here and remove the hardcode logic?

Reply:
Those are currently hard-coded in [`queryTemplate`][1].  Plus one to being consistent about hard-coding vs. configuring for these parameters, instead of handling some labels one way and some another.

[1]: https://github.com/openshift/patchmanager/blob/f266bd3279df97f7bce602d5d99346bed254b1e5/pkg/github/lister.go#L16-L17

Reply:
Rolling out the config to set up the new labels and teach teams that they need to add them will take some time. I want, for now, to have the tool explicitly comment on PRs that don't have the new label as another way to communicate that. Eventually we'll configure the tool to ignore the label and Prow to require it.

Reply:
> Eventually we'll configure the tool to ignore the label and Prow to require it.

i think we'll still want the tool to respect the risk-assed label in the future.  If a PR doesn't have "lgtm, approved, and risk-assessed" then we don't want anyone who's assessing PRs(presumably this is QE at some point) for final CPA to look at it.

But for now i understand why you want to distinguish between missing lgtm/approve(ignore PR), and missing risk-assessed(comment on PR).